amp.php

```diff
@@ -131,9 +131,9 @@ function amp_maybe_add_actions() {
 	// Add hooks for when a themes that support AMP.
 	if ( current_theme_supports( 'amp' ) ) {
 		if ( amp_is_canonical() || is_amp_endpoint() ) {
-			AMP_Theme_Support::register_hooks();
+			AMP_Theme_Support::init();
 		} else {
-			AMP_Frontend_Actions::register_hooks();
+			amp_add_frontend_actions();
```

Comment:
Why calling this here again since it is already a `wp_head` callback and it has a conditional statement only to load in paired mode?

Reply:
@ThierryA I don't understand. This function is only being called here in the `amp_maybe_add_actions()` function. It gets called here at the top in the case where the theme supports `amp`. Otherwise, it gets called below when the theme does not.